resources/definitions/fdmprinter.def.json

```diff
@@ -2587,7 +2610,7 @@
                     "type": "float",
                     "default_value": 60,
                     "value": "default_material_bed_temperature",
-                    "resolve": "max(extruderValues('material_bed_temperature'))",
+                    "resolve": "max(extruderValues('material_bed_temperature', where='extruder_used'))",
```

Comment:
Keyword arguments are not supported in Curator. We will need to extend the parser to support this.